src/js/hopscotch.js

```diff
@@ -2387,4 +2405,8 @@
 // @@include('../../tmp/js/hopscotch_templates.js') //
 }());
```

Comment:
I believe we could get around swapping out window.hopscotch with a couple of changes...
- Use `(function(){...}).call(winHopscotch);`
- Update the namespace for the `jst` task in `Gruntfile.js` to use `templates` instead of `hopscotch.templates`.